pandas/tests/series/test_constructors.py

```diff
@@ -1447,6 +1447,12 @@ def test_constructor_dict_of_tuples(self):
         expected = Series([3, 6], index=MultiIndex.from_tuples([(1, 2), (None, 5)]))
         tm.assert_series_equal(result, expected)
 
+        # GH 60695
+        data = {(1,): 3, (4, 5): 6}
+        result = Series(data).sort_values()
+        expected = Series([3, 6], index=MultiIndex.from_tuples([(1, None), (4, 5)]))
+        tm.assert_series_equal(result, expected)
```

Comment:
Can you parameterize this test instead. Having multiple tests in the same function one after the other can make debugging harder.